Tests/BackendIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -935,6 +935,7 @@ class StoreKit1IntegrationTests: BaseStoreKitIntegrationTests {
     }
 
     func testVerifyPurchaseGrantsEntitlementsThroughOnRetryAfter429() async throws {
+        defer { HTTPStubs.removeAllStubs() }
```

Comment:
should this just be a setup or teardown?

Reply:
Moved it to the `tearDown` functions 😎